azkaban-common/src/main/java/azkaban/executor/OnContainerizedExecutionEventListener.java

```diff
@@ -76,12 +76,10 @@ void restartExecutableFlow(final ExecutableFlow originalExFlow) {
         try {
           disableSucceededSkippedJobsInRetryFlow(originalExFlow, retryExFlow);
         } catch (ExecutorManagerException e){
-          // rebuild the execution in case data-structure already messed-up
-          retryExFlow = this.executorManagerAdapter.createExecutableFlow(project, flow);
-          logger.warn(String.format(
-              "fail to apply %s restart-strategy for the execution %s, fallback to default method",
-                  FlowRetryStrategy.DISABLE_SUCCEEDED_NODES.name(),
-                  originalExFlow.getExecutionId()),
+          logger.error(String.format(
+              "Fail to restart execution %s due to error applying %s restart-strategy",
+                  originalExFlow.getExecutionId(), 
+                  FlowRetryStrategy.DISABLE_SUCCEEDED_NODES.name()),
```

Comment:
Should we notify users that Azkaban encountered an error while retrying the flow execution? If yes, it can be addressed in a separate PR but can we please add a TODO in the code for now?

Reply:
I don't have a good idea now, so let me add TODO and note in the design doc